frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -863,6 +864,17 @@ final class BloopBspServices(
   private def toBuildTargetId(project: Project): bsp.BuildTargetIdentifier =
     bsp.BuildTargetIdentifier(project.bspUri)
 
+  private def toJvmBuildTarget(project: Project): bsp.JvmBuildTarget = {
+    project.jdkConfig
```

Comment:
Nit: could we make all of this as concise as possible by using pattern matching here ?

Reply:
ok